test/opendss.jl

```diff
@@ -54,10 +54,23 @@
         Memento.setlevel!(TESTLOG, "info")
 
         @test_throws(TESTLOG, ErrorException,
-                     TPPMs.parse_file("../test/data/opendss/test_simple3.dss"))
+                   TPPMs.parse_file("../test/data/opendss/loadparser_error_1.dss"))
```

Comment:
Could we perhaps create a subfolder to deal with the clutter for now? 

What you propose would be helpful here and in other cases.  A method could be added which takes a template dss file, and inserts a specified string at a certain position. This could be used to create new dss files on the fly, and run them through the parser in the tests.

Reply:
I rewrote the tests as loops instead, as you suggested.